src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -121,15 +121,35 @@ const DropdownMenuItem: React.FC<T.ItemProps> = (props) => {
 };
 
 const DropdownMenuSubMenu: React.FC<T.SubMenuProps> = (props) => {
-	const { children } = props;
+	const {
+		children,
+		position = "end-top",
+		forcePosition,
```

Comment:
I think we can spread everything except for children and default values as ...dropdownMenuProps, similar to how it's already done in other flyout components. That way we wont have to update the implementation each time a new prop is added

Reply:
updated